lib/trento/software_updates/discovery.ex

```diff
@@ -41,7 +41,16 @@ defmodule Trento.SoftwareUpdates.Discovery do
     do:
       {:ok,
        Hosts.get_all_hosts()
-       |> Enum.map(&discover_host_software_updates/1)
+       |> Enum.map(fn
```

Comment:
Nothing for this PR, but I find having such a long functions as one liners a bit ugly hehe
Even though it is a piped one lines, I would definitely add a `do ... end` block.

Reply:
you mean the function in the `map` or the whole `discover_software_updates`?

Reply:
I guess `discover_software_updates`. Fixed :smile:

Reply:
Yes! thank you!